electro

```diff
@@ -1,19 +1,19 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
sconsiglio di utilizzare `bin/bash`, perché meno "corretto".
utilizzando `bin/bash` verrà utilizzato quel comando, con `/usr/bin/env bash` verrà cercato `bash` nel proprio `$PATH` utilizzando quindi la versione corretta.
immagina di aggiornare bash tramite **brew**, il tuo eseguibile sarebbe in `/usr/local`